src/main/resources/db/changelog/changelog-1.yaml

```diff
@@ -0,0 +1,59 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: Victor
+      preConditions:
+        onFail: MARK_RAN
```

Comment:
la ce foloseste acest MARK_RAN

Reply:
din ce am citit asta se foloseste in cazul in care tabelul exista deja in Db sa treaca mai departe si sa nu dea eroare